compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt

```diff
@@ -172,13 +175,14 @@ internal class ComposeContainer(
             "ComposeUIViewController.view should be attached to window"
         }
         val scale = window.screen.scale
```

Comment:
Shouldn't we drop accessing the winding here?

Reply:
What do you mean?
Should I remove window usage from here?

Reply:
Done